### ISSUE TITLE
Don't show keyboard when showing tags

### ISSUE THREAD

**Issue by ltGuillaume** [TBog/TBLauncher#282]
**Description**

When showing tags, or the tags menu as result, it would make sense to _not_ open the keyboard:
- When you're using tags this way, you're browsing a list, not searching for a specific result
- You can't search-by-type _within_ tag results anyway: it will show overall search results

**Solution**

Don't open keyboard.

**Alternative solutions**

Allow searching within the currently chosen tag only.

**Comment by victor0d** [comment]
Hi Guillaume,
I use the new `Show Search Desktop with keyboard` feature for this. I am not sure if it would work for you, but for me it is perfect.
Obviously for that to work, you need to disable `Link Keyboard to Search bar visibility`.
This way when I go from Widgets to Search, depending on my choice, I get the keyboard active or not (i.e. searching or displaying tags/menus)

**Comment by TBog** [comment]
I'm not willing to do this if link keyboard with search bar is on.

I understand your logic and I agree, but if this is a bug, can you please provide the settings you are using and the steps needed to reproduce the problem?

**Comment by ltGuillaume** [comment]
Ah, thx, I didn't see that action implemented yet, cool! 

I can work with that, as long as the "Back action when closing keyboard" would work again. Strangely, that seems to stop working altogether here (on 6.11), even when using the Smart Task Launcher "Back" action via Android's accessibility feature.

**Comment by ltGuillaume** [comment]
Changing everything to Show Search Desktop with keyboard I did find that it doesn't work under "Initial Desktop" yet. Just choose "Close launcher" and the keyboard won't pop up after the restart.

**Comment by victor0d** [comment]
Hi Guillaume,
I believe these three cases that you are describing, are actually the same case: the launcher is re-starting from a different activity (in the last two cases, the home and back presses are processed in the "other" app environment, rather than the launcher).
The feature  `Show Search Desktop with keyboard` seems working as expected.
But what seems to me not working as expected, is the feature `Behaviour` `Initial desktop` `Desktop Search with keyboard`.  
I don't use this feature, but it seems the one you are describing.

**Comment by ltGuillaume** [comment]
> I believe these three cases that you are describing, are actually the same case: the launcher is re-starting from a different activity

Yep, Ikd say that's exactly what's happening (see the first line of my reply). Those 3 situations all seem to lead to that conclusion.

**Comment by TBog** [comment]
commit d8c93bdf3c3101ccf1d7114d72de2bbe67535930 may fix "launcher is re-starting from a different activity" issue

**Comment by ltGuillaume** [comment]
Unfortunately, it doesn't completely fix the issue: the keyboard often doesn't show when getting back into the launcher from another app.

**Comment by TBog** [comment]
Does [v6.15-dev.1](https://github.com/TBog/TBLauncher/releases/tag/v6.15-dev.1) fix the keyboard not showing issue?

**Comment by ltGuillaume** [comment]
Will check in the upcoming days. Thanks!
